llvm/include/llvm/Analysis/TargetLibraryInfo.h

```diff
@@ -408,35 +408,42 @@ class TargetLibraryInfo {
     switch (F) {
     default: break;
       // clang-format off
-    case LibFunc_copysign:     case LibFunc_copysignf:  case LibFunc_copysignl:
-    case LibFunc_fabs:         case LibFunc_fabsf:      case LibFunc_fabsl:
-    case LibFunc_sin:          case LibFunc_sinf:       case LibFunc_sinl:
-    case LibFunc_cos:          case LibFunc_cosf:       case LibFunc_cosl:
-    case LibFunc_tan:          case LibFunc_tanf:       case LibFunc_tanl:
-    case LibFunc_asin:         case LibFunc_asinf:      case LibFunc_asinl:
     case LibFunc_acos:         case LibFunc_acosf:      case LibFunc_acosl:
+    case LibFunc_acosh:        case LibFunc_acoshf:     case LibFunc_acoshl:
```

Comment:
would it be possible to avoid the re-formatting? This makes it very difficult to see what gets added (and it looks like clang-format is disabled for this section intentionally)

Reply:
I'll split it into a NFC patch that sorts the table, followed by the additions, and then pre-land the NFC patch.

Reply:
Thanks! Can you add tests for a few more of the newly added ones, maybe one per family (pow/erf/pow/cbrt)?